### PR TITLE
Skip Flakey Tests

### PR DESCRIPTION
**Pull request by paroma**
* Set seed for LabelModel docs
* Increase delta for checking equality to account for rounding issues
* Skip flakey tests

### CODE REVIEW

**Comment by henryre** [comment]
@paroma what's the status here? we at least need to skip the non-deterministic ones to unblock people (got a tox failure for the [0.99, 0.99, 0.99] doctest or whatever)

File: snorkel/labeling/model/label_model.py [hunk 1]
```diff
@@ -366,9 +366,9 @@ def predict_proba(self, L: np.ndarray) -> np.ndarray:
 
         Example
         -------
-        >>> L = np.array([[0, 0, 0], [1, 1, 1], [1, -1, 1]])
+        >>> L = np.array([[0, 0, 0], [1, 1, 1], [1, 1, 1]])
```
Comment (henryre) [inline snorkel/labeling/model/label_model.py]: Probably a worse example without an abstain. In general, this L is just not super realistic.

File: snorkel/labeling/model/label_model.py [hunk 2]
```diff
@@ -340,7 +340,7 @@ def get_accuracies(self) -> np.ndarray:
         -------
         >>> L = np.array([[1, 1, 1], [1, 1, -1], [-1, 0, 0], [0, 0, 0]])
         >>> label_model = LabelModel(verbose=False)
-        >>> label_model.fit(L)
+        >>> label_model.fit(L, n_epochs=1000, seed=123)
```
Comment (henryre) [inline snorkel/labeling/model/label_model.py]: Is there any other source of randomness when we set this seed? If not, we can drop the number of epochs and make the Ls more realistic.

**Comment by paroma** [comment]
I commented out the doctests for now, revisiting post-release
